### ISSUE TITLE
version on git

### ISSUE THREAD

**Issue by sh2515** [silvanmelchior/RPi_Cam_Web_Interface#183]
Hi

Would you be able to have the latest version number in the title of the web page please? Just a quick way that helps me know when I should be updating.

**Comment by roberttidey** [comment]
OK will check into that.

**Comment by roberttidey** [comment]
The main web page did have version in, and I've added that to the other web pages now.

**Comment by sh2515** [comment]
Yes it was to do with this webpage on git. Maybe a release number on the readme file or on one of the titles. Maybe even akin to the style on motionpie where the folders are named year month revision.
